tests/kernel/qa_test_klp.pm

```diff
@@ -30,15 +32,13 @@ sub run {
     # Set and check patch variables
     my $incident_id = get_var('INCIDENT_ID');
     my $patch       = get_var('INCIDENT_PATCH');
-    check_patch_variables($patch, $incident_id);
+    check_patch_variables($patch, $incident_id) if (!get_var('BETA'));
 
-    select_console('root-console');
+    is_sle(">12-sp1") ? $self->select_serial_terminal() : select_console('root-console');
```

Comment:
isn't the serial terminal availability only depending on the version of qemu we run on our workers?

Reply:
@okurz IMHO it's not a worker issue, it's a SUT issue. On old SLES it's needed to add `hvc1` into `/etc/securetty` and enable `serial-getty@hvc1`, which is what `add_serial_console('hvc1')` does. @mimi1vx prefers using non-virtio console to avoid problems.

Reply:
@okurz I mean @mimi1vx from QAM does reviewing of it, so I respect his requests for it.